db/migrate/20160323171249_remove_deleted_from_attachments.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDeletedFromAttachments < ActiveRecord::Migration
+  def change
+    remove_column :attachments, :deleted, :boolean, null: false, default: false
```

Comment:
This can't be run as part of this PR, the code that no longer references it (along with `deprecated_columns`) needs to be deployed first, then this run as a separate deployment.